utils/newLesson.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+// This little script can be used to create a new skeleton lesson
+
+const fs = require('fs');
+
+function getMaxLesson() {
+    return fs.readdirSync('.').reduce((maxLesson, file) => {
+        const match = file.match(/lesson(\d+).md/);
+        if (match === null) {
+            return maxLesson;
+        }
+        return Math.max(maxLesson, parseInt(match[1]));
+    }, 0);
+}
+
+function getNextDate() {
+    const date = new Date();
+    if (date.getDay() === 3 || date.getDay() === 4) {
+        date.setDate(date.getDate() + 4 - date.getDay());
+    } else {
+        date.setDate(date.getDate() + (2 + 7 - date.getDay()) % 7);
+    }
+    return date;
```

Comment:
having a really hard time trying to figure out how this actually works, but it does : D

Reply:
Could be made easier to read with momentjs: https://stackoverflow.com/questions/19100373/moment-js-getting-next-date-given-specified-week-day/19100624

But then you'd need that installed globally since this isn't an npm package